src/stylesheets/base/_colors.scss

```diff
@@ -0,0 +1,14 @@
+$colors: (
+  cool-charcoal:        #252224,
+  warm-charcoal:        #504949,
+  deep-blue:            #184D66,
+  emerald:              #0E5254,
+  teal:                 #1793AD,
+  mint:                 #D3E7E2,
+  grey:                 #E2EBE9,
+  white:                #FFFFFF,
+  yellow:               #F2C84D,
+  orange:               #C06D3A,
+  coral:                #EA4446,
+  gradient-background:  linear-gradient(#184D66 3.08%, #D3E7E2 94.99%)
```

Comment:
one thing i like to do to keep things real clear is to use the base color in the name so its clear what it is when you see it in the wild. for example `green-mint` or `pink-coral`

Reply:
Noted.